### PR TITLE
Deleted

### PR DESCRIPTION
**Pull request by vextrixes**
Deleted

### CODE REVIEW

File: indicators/roblox-phishing.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+title: Roblox phishing
+description: |
+    Detects Roblox phishing sites using a roblox body id and cdn.
+    Usually at /controlPage/create you can create a "Beaming link".
+    Often spread trough discord.
+
+level: likely_malicious
```
Comment (IlluminatiFish) [inline indicators/roblox-phishing.yml:7]: The use of `level` in the public ruleset can be omitted.

File: indicators/roblox-phishing.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
+title: Roblox phishing
```
Comment (IlluminatiFish) [inline indicators/roblox-phishing.yml:1]: Please follow the rule naming convention specified in `CONTRIBUTING.md`. Typically the rule title and file name are the same.

File: indicators/roblox-phishing.yml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    Detects Roblox phishing sites using a roblox body id and cdn.
+    Usually at /controlPage/create you can create a "Beaming link".
+    Often spread trough discord.
+
+level: likely_malicious
+
+references:
+  - https://www.youtube.com/watch?v=lUL2vgyhsw4
+  - https://urlscan.io/result/c716b820-174e-4211-9c09-4663b4a7e47d/
+  - https://urlscan.io/result/c716b820-174e-4211-9c09-4663b4a7e47d/
```
Comment (IlluminatiFish) [inline indicators/roblox-phishing.yml:12]: It seems you have a duplicate reference here.

**Review by IlluminatiFish — CHANGES_REQUESTED** [review]
Please make the requested changes.

**Comment by Czdam0** [comment]
Thank you for bringing these issues to my attention. I have fixed these issues.

**Review by vextrixes — COMMENTED** [review]
Deleted

**Review by IlluminatiFish — APPROVED** [review]

**Pull request merged**